The resourceManager module has a compatibility bug when Yarn is in capacity scheduler mode

**Describe the bug**
The resourceManager module has a wrong conversion when Yarn is in the capacity scheduler mode to obtain the maximum available resources of the Yarn queue.

**Expected behavior**
There is an incorrect conversion from JDouble to JDecimal, and the error log is as follows
```
Caused by: java.lang.ClassCastException: org.json4s.JsonAST$JDouble cannot be cast to org.json4s,JsonAST$JDecimal
```

--- a/linkis-computation-governance/linkis-manager/linkis-resource-manager/src/main/scala/com/webank/wedatasphere/linkis/resourcemanager/external/yarn/YarnResourceRequester.scala
+++ b/linkis-computation-governance/linkis-manager/linkis-resource-manager/src/main/scala/com/webank/wedatasphere/linkis/resourcemanager/external/yarn/YarnResourceRequester.scala
@@ -62,7 +62,23 @@ class YarnResourceRequester extends ExternalResourceRequester with Logging {
       val metrics = getResponseByUrl( "metrics", rmWebAddress)
       val totalResouceInfoResponse = ((metrics \ "clusterMetrics" \ "totalMB").asInstanceOf[JInt].values.toLong, (metrics \ "clusterMetrics" \ "totalVirtualCores").asInstanceOf[JInt].values.toLong)
       queueValue.map(r => {
-        val effectiveResource = (r \ "absoluteCapacity").asInstanceOf[JDouble].values - (r \ "absoluteUsedCapacity").asInstanceOf[JDouble].values
+        val absoluteCapacity = r \ "absoluteCapacity" match {
+          case jDecimal: JDecimal =>
+            jDecimal.values.toDouble
+          case jDouble: JDouble =>
+            jDouble.values
+          case _ =>
+            0d
+        }
+        val absoluteUsedCapacity =  r \ "absoluteUsedCapacity" match {
+          case jDecimal: JDecimal =>
+            jDecimal.values.toDouble
+          case jDouble: JDouble =>
+            jDouble.values
+          case _ =>
+            0d
+        }
+        val effectiveResource = absoluteCapacity - absoluteUsedCapacity
         new YarnResource(math.floor(effectiveResource * totalResouceInfoResponse._1 * 1024l * 1024l/100).toLong, math.floor(effectiveResource * totalResouceInfoResponse._2/100).toInt, 0, queueName)
       })
     }

